app/models/cafe_work.rb

```diff
@@ -0,0 +1,233 @@
+    end
+  end
+
+  # User to update worker, checks for edit-access and triggers at_update
+  # /d.wessman
+  def add_worker(worker_params, user)
+    if has_worker?
+      errors.add('Arbetare', 'passet har redan en.')
+      return false
+    end
+
+    # Should be done with a bang when the error handling works
+    # Ref: https://github.com/fsek/web/issues/93
+    # /d.wessman
+    #update!(Assignee.setup(worker_params, user).attributes)
+    update(Assignee.setup(worker_params, user).attributes)
+  end
+
+  # User to update worker, checks for edit-access
+  # /d.wessman
+
+  def update_worker(worker_params, user)
+    if (!owner?(user) && !authorize(worker_params[:access_code]))
+      errors.add('Auktorisering', 'misslyckades, du har inte rättighet att redigera eller skrev fel kod.')
+      return false
+    end
+
+    # Should be done with a bang when the error handling works
+    # Ref: https://github.com/fsek/web/issues/93
+    # /d.wessman
+    #update!(Assignee.setup(worker_params, user).attributes)
+    update(Assignee.setup(worker_params, user).attributes)
+  end
+
+  # Remove-function used by the worker
+  # /d.wessman
+  def remove_worker(user, access)
+    if (!owner?(user) && !authorize(access))
+      errors.add('Auktorisering', 'misslyckades, du har inte rättighet att ta bort eller skrev fel kod.')
+      return false
+    end
+
+    clear_worker
+  end
+
+  # Method to remove the worker from current work.
+  # /d.wessman
+  def clear_worker
+    self.attributes = worker.clear_attributes
+    self.utskottskamp = false
+    self.councils.clear
+    return self.save!(validate: false)
+  end
+
+  # Returns true if the profiles are similar and not nil
+  # /d.wessman
+  def owner?(user)
+    user.present? && user.profile.present? && profile.present? && user.profile == profile
+  end
+
+  # Returns true if the user can edit the object
+  # /d.wessman
+  def edit?(user)
+    editable? && (!has_worker? || owner?(user))
+  end
+
+  def editable?
+    work_day > Time.zone.now
+  end
+
+  # Returns true only if hte access_code is correct
+  # /d.wessman
+  def authorize(access)
+    access.present? && access_code.present? && access_code == access
+  end
+
+  # Returns true if there is a worker
+  # /d.wessman
+  def has_worker?
+    worker.is_present?
+  end
+
+  # Used to print date in a usable format.
+  # /d.wessman
+  def print_time
+    %(#{work_day.strftime("%H:%M")}-#{(work_day + duration.hours).strftime("%H:%M")})
+  end
+
+  # Used to print out date, reading week and work number
+  # /d.wessman
+  def print
+    %(#{print_date} LV: #{lv} Pass: #{pass})
+  end
+
+  def print_date
+    %(#{print_time}, #{work_day.strftime("%A %d/%m")})
+  end
+
+  # Prints the url or path for the current object
+  def p_url
+    Rails.application.routes.url_helpers.cafe_work_url(id, host: PUBLIC_URL)
+  end
+
+  def p_path
+    Rails.application.routes.url_helpers.cafe_work_path(id)
+  end
+
+  def as_json(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `as_json(*)` if you want the method to accept any arguments but don't care about them.